proposals/lambda-method-group-defaults.md

```diff
@@ -1,15 +1,19 @@
-# Default Parameters for Lambdas and Method Groups
+# Default and Variable Parameters for Lambdas and Method Groups
```

Comment:
```suggestion
# Default and Parameter Array Parameters for Lambdas and Method Groups
```

Reply:
What about "Optional and Parameter Array Parameters for Lambdas and Method Groups" (since "default parameters" are not a thing https://github.com/dotnet/csharplang/pull/6542#discussion_r994975532)? 3351f2725b0b524cd9d7000cfbd8d96d5b0c82be. And wouldn't "Optional and `params` Array Parameters for Lambdas and Method Groups" be even better?

Also, should I change the filename (currently `lambda-method-group-defaults.md`)?